packages/synapse-react-client/src/components/widgets/SynapseImage.tsx

```diff
@@ -25,123 +25,72 @@ export type SynapseImageProps = {
   }
 }
 
-type SynapseImageState = {
-  preSignedURL: string
-}
-
-class SynapseImage extends Component<SynapseImageProps, SynapseImageState> {
-  constructor(props: SynapseImageProps) {
-    super(props)
-    this.getEntity = this.getEntity.bind(this)
-    this.getSynapseFiles = this.getSynapseFiles.bind(this)
-    this.state = {
-      preSignedURL: '',
-    }
-  }
+function SynapseImage(props: SynapseImageProps) {
```

Comment:
convert to function component and use existing `ImageFileHandle` component to handle rendering the image - this becomes very thin and just 1) constructs the FileHandleAssociation and 2) maps markdown properties onto the element